pkg/agent/baker.go

```diff
@@ -787,5 +787,8 @@ func getContainerServiceFuncMap(config *datamodel.NodeBootstrappingConfiguration
 		"HasMessageOfTheDay": func() bool {
 			return profile.MessageOfTheDay != ""
 		},
+		"IsUnattendedUpgradeEnabled": func() bool {
+			return config.IsUnattendedUpgradeEnabled
```

Comment:
can't start with this defaulted to false or have to change rp when you vendor it in which is risky